code/game/objects/items/storage/holsters.dm

```diff
@@ -115,7 +115,6 @@
 		/obj/item/ammo_magazine/rocket,
 		/obj/item/weapon/gun/launcher/rocket/recoillessrifle,
 	)
-	sprite_sheets = list("Combat Robot" = 'icons/mob/species/robot/backpack.dmi') //robots have their own snowflake back sprites
```

Comment:
?

Reply:
Ah balls, I left that out because species sprites were broken at the time I started this PR. I'll sort this out. Bloody robots.